pkg/registry/apis/identity/display.go

```diff
@@ -0,0 +1,191 @@
+package identity
+
+import (
+	"context"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/grafana/authlib/claims"
+	"github.com/grafana/grafana/pkg/api/dtos"
+	identity "github.com/grafana/grafana/pkg/apimachinery/apis/identity/v0alpha1"
+	"github.com/grafana/grafana/pkg/registry/apis/identity/legacy"
+	"github.com/grafana/grafana/pkg/services/apiserver/endpoints/request"
+	"github.com/grafana/grafana/pkg/setting"
+	errorsK8s "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apiserver/pkg/registry/rest"
+)
+
+type displayREST struct {
+	store legacy.LegacyIdentityStore
+}
+
+var (
+	_ rest.Storage              = (*displayREST)(nil)
+	_ rest.SingularNameProvider = (*displayREST)(nil)
+	_ rest.Connecter            = (*displayREST)(nil)
+	_ rest.Scoper               = (*displayREST)(nil)
+	_ rest.StorageMetadata      = (*displayREST)(nil)
+)
+
+func newDisplayREST(store legacy.LegacyIdentityStore) *displayREST {
+	return &displayREST{store}
+}
+
+func (r *displayREST) New() runtime.Object {
+	return &identity.IdentityDisplayResults{}
+}
+
+func (r *displayREST) Destroy() {}
+
+func (r *displayREST) NamespaceScoped() bool {
+	return true
+}
+
+func (r *displayREST) GetSingularName() string {
+	return "QueryResults" // Used for the
```

Comment:
I feel like this is not the full comment. What is "QueryResults" in this context?